src/sider/arbeidsuforhet/SendForhandsvarsel84Skjema.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { addWeeks } from "@/utils/datoUtils";
+import { ButtonRow } from "@/components/Layout";
+import { Button, Textarea } from "@navikt/ds-react";
+import { useForm } from "react-hook-form";
+import { SkjemaHeading } from "@/sider/aktivitetskrav/vurdering/SkjemaHeading";
+import { useArbeidsuforhetVarselDocument } from "@/hooks/arbeidsuforhet/useArbeidsuforhetVarselDocument";
+import { Forhandsvisning } from "@/components/Forhandsvisning";
+
+const texts = {
+  title: "Send forhåndsvarsel",
+  beskrivelseLabel: "Begrunnelse (obligatorisk)",
+  begrunnelseDescription:
+    "Begrunnelsen din blir en del av en større brevmal. Åpne forhåndsvisning for å se hele varselet.",
+  forhandsvisning: "Forhåndsvisning",
+  forhandsvisningLabel: "Forhåndsvis forhåndsvarselet",
+  missingBeskrivelse: "Vennligst angi begrunnelse",
+  sendVarselButtonText: "Send",
+  avbrytButtonText: "Avbryt",
+  malLabel: "Velg arbeidssituasjon",
+  malHelptext:
+    "Her kan du velge mellom ulike brevmaler som er tilpasset den sykmeldtes arbeidssituasjon",
+};
+
+const forhandsvarselFrist = addWeeks(new Date(), 3);
+const defaultValues = { begrunnelse: "" };
+const begrunnelseMaxLength = 1000;
```

Comment:
<img src="https://media3.giphy.com/media/YmQLj2KxaNz58g7Ofg/giphy.gif"/>
Kanskje vi skal følge med på om dette blir et problem for akkurat denne. Kan tenke seg at det krever litt mer dokumentasjon og forklaring? Selv om man kommer langt med 1000 tegn da.